### PR TITLE
Add a description for a form of type Bellow pages [MAILPOET-3159]

### PR DESCRIPTION
**Pull request by lysyjan**
[MAILPOET-3159]

[MAILPOET-3159]: https://mailpoet.atlassian.net/browse/MAILPOET-3159

### CODE REVIEW

File: views/form/editor.html [hunk 1]
```diff
@@ -93,6 +93,7 @@
   'exitIntentDescription': __('Show the form immediately if the visitor attempts to leave the site.'),
   'exitIntentSwitch': __('Exit-intent display'),
   'placeFormBellowPages': _x('Below pages', 'This is a text on a widget that leads to settings for form placement'),
+  'placeFormBellowPagesPagesDescription': __('Display your form at the bottom of posts or pages.'),
   'placeFormBellowPagesDescription': __('This form placement allows you to add this form at the end of all the pages or posts, below the content.'),
```
Comment (costasovo) [inline views/form/editor.html:97]: I can see that we already have a translation for bellow pages description. Just take a look one line below. We probably accidentally removed it from the react component during refactoring.
I guess we could reuse the old translation key, which seems a bit nicer than the new one with double "pages". Maybe we could also reuse the older wording as is. The older text has been probably already translated. You can ask @veljkho about it.

Comment (lysyjan) [reply views/form/editor.html:97]: You are right. I'm feeling really blind at this moment. I'm sorry.

Comment (costasovo) [reply views/form/editor.html:97]: We have about a hundred keys there so it is easy to miss one and you didn't know we used to have some description. So you don't need to be sorry.

Comment (lysyjan) [reply views/form/editor.html]: We decided to use the old description.

**Review by costasovo — APPROVED** [review]

**Review by veljkho — APPROVED** [review]

**Pull request merged**